app/templates/index.hbs

```diff
@@ -1,4 +1,2 @@
- <main>
-    <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
-    <h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events">here</a></h2>
-</main>
\ No newline at end of file
+<h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
+<h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events">here</a></h2>
```

Comment:
While href will work, Ember has it's specific way of [linking](https://guides.emberjs.com/release/routing/linking-between-routes/) 
The reason why an href in place of a LinkTo is frowned upon is because href will reload your entire page, in essence reloading your entire ember application, for simple routing
LinkTo has been made in way where it routes you to the desired URL without reloading the entire ember application (which is an SPA)